DbcParserLib/Model/MultiplexingRole.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace DbcParserLib.Model
+{
+    public enum MultiplexingRole
+    {
+        None, 
+        Unknown, // Used if parsing fails
+        Multiplexed, 
+        Multiplexor, 
+        MultiplexedMultiplexor
```

Comment:
Or should mark `MultiplexingRole` as `[Flags]` and have a multiplexed multiplexor be a `Multiplexed | Multiplexor`?
I have mixed feelings: one one hand it makes easier to spot how is muxed and who is muxer, on the other hand we should remove the `Unknown` value because it can't be mixed.